app/models/dialog_field_serializer.rb

```diff
@@ -23,7 +23,7 @@ def serialize(dialog_field, all_attributes = false)
 
     if dialog_field.type == "DialogFieldTagControl"
       category = Category.find_by(:id => dialog_field.category)
-      dialog_field.options.merge!(:category_name => category.name, :category_description => category.description)
+      dialog_field.options.merge!(:category_name => category.name, :category_description => category.description) if category.present?
```

Comment:
The `present?` shouldn't be necessary since `find_by` will return either a record or nil